src/scene/scene_lua.inl

```diff
@@ -28,73 +28,115 @@ namespace openspace {
 
 namespace {
 
-void applyRegularExpression(lua_State* L, std::regex regex, std::vector<properties::Property*> properties, int type) {
+void executePropertySet(properties::Property* prop, lua_State* L) {
+    prop->setLuaValue(L);
+    //ensure properties are synced over parallel connection
+    std::string value;
+    prop->getStringValue(value);
+    /*OsEng.parallelConnection().scriptMessage(
+    prop->fullyQualifiedIdentifier(),
+    value
+    );*/
+}
+
+template <class T>
+properties::PropertyOwner* findPropertyOwnerWithMatchingGroupTag(T* prop,
```

Comment:
Coding style:  Try to put the `tagToMatch` on the same line, if it fits.. otherwise:
```
properties::PropertyOwner* findPropertyOwnerWithMatchingGroupTag(T* prop,
                                                                                                                   const std::string& tagToMatch)
{
```

in either way: `const std::string&`